.gitignore

```diff
@@ -22,3 +22,5 @@ go.work
 
 # Build file
 /build
+client.http
```

Comment:
Can you share more details about this file?

Reply:
Oh, I created it for making http request. It is very convenient. If you use vscode, you can install RestClient and it is enought to run client.http.

Reply:
Oh, Thank you! I get the new useful tool thanks to you! 😍